brands/amenity/ice_cream.json

```diff
@@ -344,6 +344,34 @@
       "takeaway": "yes"
     }
   },
+  "amenity/ice_cream|smöoy": {
+    "countryCodes": [
+      "ao",
+      "ci",
+      "cn",
+      "dz",
+      "ec",
+      "es",
+      "fr",
+      "gn",
+      "it",
+      "ma",
+      "pa",
+      "pe",
+      "pt",
+      "re",
+      "sa",
+      "uk"
```

Comment:
Should this be gb country code for the UK?

Reply:
You're right, it's my mistake. Why is United Kingdoms's code GB if they're going to reserve UK anyways? It's confusing.

Fixed with commit 75d1606ea19d0c7b3964743ff8bd5e1433b7f115. I also fixed it in other entries; some of them were opened by me, others weren't.